tests/Feature/CelebrityTest.php

```diff
@@ -44,4 +45,56 @@ public function test_admin_can_access_celebrities_create_page()
                 ->get('/celebrities/create')
                 ->assertOk();
     }
+
+    /** @test  */
+    public function test_admin_can_access_celebrities_edit_page()
+    {
+        $admin = User::factory()->create([
+            'is_admin' => true
+        ]);
+        $celebrity = Celebrity::factory()->create();
+
+        $this->actingAs($admin)
+                ->get('/celebrities/'.$celebrity->id.'/edit')
+                ->assertOk();
+    }
+
+    /** @test  */
+    public function test_users_cannot_access_celebrities_edit_page()
+    {
+        $user = User::factory()->create([
+            'is_admin' => false
+        ]);
+        $celebrity = Celebrity::factory()->create();
+
+        $this->actingAs($user)
+                ->get('/celebrities/'.$celebrity->id.'/edit')
+                ->assertRedirect('/');
+    }
+
+    /** @test  */
+    public function test_users_cannot_delete_celebrities()
+    {
+        $user = User::factory()->create([
+            'is_admin' => false
+        ]);
+        $celebrity = Celebrity::factory()->create();
+
+        $this->actingAs($user)
+                ->delete('/celebrities/'.$celebrity->id)
+                ->assertRedirect('/');
```

Comment:
Hey Libro

Whenever we are testing something that might change the DB, we also need to check if the DB is as expected. 

You can do that by using `assertDatabaseHas` or `assertDatabaseMissing` in the case of an unauthorized delete action like this one. We want o check if the record is still on the Database. So you would use the `assertDatabaseHas` to do that. 

If an admin deletes a record, we want to ensure that the record has been soft deleted in the database (that is if the class allows soft deletes) 😄